src/images/icons/chevron-down-grey.svg

```diff
@@ -0,0 +1,3 @@
+<svg height="20" width="20" viewBox="0 0 20 20" fill="grey">
```

Comment:
I am fine with this, but we could style the svg dynamically, rather than swapping the assets themselves